pkg/cli/alpha/internal/update/utils.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package update
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"runtime"
+)
+
+const releaseURL = "https://github.com/kubernetes-sigs/kubebuilder/releases/download/%s/kubebuilder_%s_%s"
+
+func buildReleaseURL(version string) string {
+	return fmt.Sprintf(releaseURL, version, runtime.GOOS, runtime.GOARCH)
+}
+
+type releaseResponse struct {
+	TagName string `json:"tag_name"`
+}
+
+func fetchLatestRelease() (string, error) {
```

Comment:
Do we use it anywhere outside of prepare?
If not, I think it makes sense to keep it there.

The utils package should only include utilities that are used in multiple places.
Could you please move it to prepare.go?

Thanks!